lib/cdo/help_header.rb

```diff
@@ -0,0 +1,79 @@
+# This class provides the content for both the hamburger and the header.
```

Comment:
is this accurate? It looks like this all unique to the help menu, not the hamburger menu

Reply:
Updated.